backend/config.py

```diff
@@ -974,10 +945,11 @@ class BannerModel(BaseModel):
 # RAG
 ####################################
 
+VECTOR_STORE_TYPE = os.environ.get("VECTOR_STORE_TYPE", "persistent_chroma")
```

Comment:
These should be moved to our new env.py file!

Reply:
Modified!